slight build problem in FreeBSD-12.1, with clang-8.0.1

```
In file included from /home/edscott/GIT/OPM/opm-upscaling/opm/upscaling/writeECLData.cpp:25:
/home/edscott/GIT/OPM/opm-upscaling/opm/upscaling/writeECLData.hpp:37:21: error: unknown type name
      'time_t'
                    time_t current_posix_time,
                    ^
1 error generated.
```

---

Fixed by adding a simple:
```C++
#include <time.h>
```

--- a/opm/upscaling/writeECLData.hpp
+++ b/opm/upscaling/writeECLData.hpp
@@ -23,6 +23,8 @@
 
 #include <string>
 
+#include <time.h>
+
 #include <opm/output/data/Solution.hpp>
 
 namespace Opm

